generators/chipyard/src/main/scala/config/ChipConfigs.scala

```diff
@@ -38,7 +65,6 @@ class ChipLikeQuadRocketConfig extends Config(
   new chipyard.config.WithFbusToSbusCrossingType(AsynchronousCrossing()) ++  // Add Async crossing between SBUS and FBUS
   new chipyard.config.WithCbusToPbusCrossingType(AsynchronousCrossing()) ++  // Add Async crossing between PBUS and CBUS
   new chipyard.config.WithSbusToMbusCrossingType(AsynchronousCrossing()) ++  // Add Async crossings between backside of L2 and MBUS
-  new testchipip.WithAsynchronousSerialSlaveCrossing ++                      // Add Async crossing between serial and MBUS. Its master-side is tied to the FBUS
```

Comment:
We may want this back later depending on what happens with clocking, but we can deal with that later